crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/panic_data.rs

```diff
@@ -66,29 +61,35 @@ mod test {
     use starknet_types_core::felt::Felt;
     use test_case::test_case;
 
-    // #[test]
-    // fn extracting_plain_panic_data() {
-    //     let cases: [(&str, Option<Vec<Felt>>); 4] = [
-    //         (
-    //             "Beginning of trace\nGot an exception while executing a hint: Hint Error: Execution failed. Failure reason: 0x434d3232 ('PANIK, DAYTA').\n
-    //              End of trace",
-    //             Some(vec![Felt::from(344_693_033_291_u64), Felt::from(293_154_149_441_u64)])
-    //         ),
-    //         (
-    //             "Got an exception while executing a hint: Hint Error: Execution failed. Failure reason: 0x434d3232 ('AYY, LMAO').",
-    //             Some(vec![Felt::from(4_282_713_u64), Felt::from(1_280_131_407_u64)])
-    //         ),
-    //         (
-    //             "Got an exception while executing a hint: Hint Error: Execution failed. Failure reason: 0x0 ('').",
-    //             Some(vec![])
-    //         ),
-    //         ("Custom Hint Error: Invalid trace: \"PANIC, DATA\"", None)
-    //     ];
-    //
-    //     for (str, expected) in cases {
-    //         assert_eq!(try_extract_panic_data(str), expected);
-    //     }
-    // }
+    #[test_case(indoc!(r"
+                    Beginning of trace
```

Comment:
Are the errors still wrapped in `Beginning of trace`, `End of trace`? I don't think this is longer a case.